Lib/test/test_base64.py

```diff
@@ -776,6 +776,20 @@ def test_b85decode_errors(self):
         self.assertRaises(ValueError, base64.b85decode, b'|NsC')
         self.assertRaises(ValueError, base64.b85decode, b'|NsC1')
 
+    def test_z85decode_errors(self):
+        illegal = list(range(33)) + \
+                  list(b'"\',;_`|\\~') + \
+                  list(range(128, 256))
+        for c in illegal:
+            with self.assertRaises(ValueError, msg=bytes([c])):
+                base64.z85decode(b'0000' + bytes([c]))
+
+        self.assertRaises(ValueError, base64.z85decode, b'%')
+        self.assertRaises(ValueError, base64.z85decode, b'%N')
+        self.assertRaises(ValueError, base64.z85decode, b'%Ns')
+        self.assertRaises(ValueError, base64.z85decode, b'%NsC')
+        self.assertRaises(ValueError, base64.z85decode, b'%NsC1')
```

Comment:
The first four should be prefixes of `b'%nSc0'` which is a Z85 encoded `b'\xff\xff\xff\xff'`. The last one should be `b'%nSc1'`, it represents 0x100000000 which cannot be packed in 4 bytes.

Perhaps it needs a comment.

Reply:
Fixed, Thanks!